pkg/operator/operator.go

```diff
@@ -165,14 +165,14 @@ func (o *Operator) RegisterMetrics(r prometheus.Registerer) {
 		Help: "Number of attempts to reconcile the operator configuration",
 	})
 
-	o.reconcileErrors = prometheus.NewCounter(prometheus.CounterOpts{
-		Name: "cluster_monitoring_operator_reconcile_errors_total",
-		Help: "Number of errors that occurred while reconciling the operator configuration",
+	o.reconcileStatus = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
Could we instead of removing `cluster_monitoring_operator_reconcile_errors_total` add this new gauge and use it in alert expression for fast recovery? Sth like:
```
rate(errors)/rate(total) > 10 AND last_status != 1
```

Reply:
pinging @simonpasquier to resolve as this expression was a suggestion from his side. i am fine with either solution. what benefits/downsides do we have compared to this?

Reply:
I'm not sure that using `cluster_monitoring_operator_reconcile_errors_total` in conjunction with `last_status` brings more benefit that using `last_status` alone?
I prefer a simpler query that is less likely to be broken.